.travis.yml

```diff
@@ -26,8 +26,9 @@ services:
 before_install:
 # NOTE: The latest version of dep requires go 1.13
 - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then mkdir -p /Users/travis/gopath/bin; fi
+- GO111MODULE=on go get sigs.k8s.io/kind
```

Comment:
Why we install kind one more time here?
We already do that in install_and_setup.sh.
BTW we should use a deterministic version e.g. `GO111MODULE=on go get sigs.k8s.io/kind@v0.5.1`

Reply:
👍 it just is required in the Make file because of another scenario/issue. Remove from here. Tks.